tools/cldr-code/src/main/java/org/unicode/cldr/test/ExampleDependencies.java

```diff
@@ -793,5 +793,21 @@ public class ExampleDependencies {
                             "//ldml/personNames/nativeSpaceReplacement",
                             "//ldml/personNames/personName[@order=\"*\"][@length=\"*\"][@usage=\"*\"][@formality=\"*\"]/namePattern",
                             "//ldml/personNames/personName[@order=\"*\"][@length=\"*\"][@formality=\"*\"]/namePattern")
+                    .putAll(
+                            "//ldml/personNames/nativeSpaceReplacement",
+                            "//ldml/personNames/personName[@order=\"*\"][@length=\"*\"][@usage=\"*\"][@formality=\"*\"]/namePattern",
+                            "//ldml/personNames/personName[@order=\"*\"][@length=\"*\"][@formality=\"*\"]/namePattern")
+
+                    // For UnicodeSet examples, need to clear this path's value if the winning value
+                    // changes
+                    .putAll(
+                            "//ldml/characters/exemplarCharacters",
+                            "//ldml/characters/exemplarCharacters")
+                    .putAll(
+                            "//ldml/characters/exemplarCharacters[@type=\"*\"]",
+                            "//ldml/characters/exemplarCharacters[@type=\"*\"]")
+                    .putAll(
+                            "//ldml/characters/parseLenients[@scope=\"*\"][@level=\"*\"]/parseLenient[@sample=\"*\"]",
+                            "//ldml/characters/parseLenients[@scope=\"*\"][@level=\"*\"]/parseLenient[@sample=\"*\"]")
```

Comment:
this file was originally auto-generated by TestExampleDependencies, and ideally we might save time by getting the auto-generation to run when needed, so we would never need to edit this file manually